### PR TITLE
Labels dex persona migration

### PR DESCRIPTION
**Pull request by Hosuke**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by jeff-dude** [comment]
blocked by `dex_aggregator.trades`, will be ready soon

**Comment by aalan3** [comment]
dex_aggregators.trades is now merged

**Comment by aalan3** [comment]
Added the remaining things for labels.dex as well here

**Comment by aalan3** [comment]
Sorry for the many commits, I couldn't get this to run properly locally

File: models/labels/addresses/dex/labels_dex.sql [hunk 1]
```diff
@@ -8,6 +9,9 @@
                                 \'["ilemi"]\') }}')
 }}
 
+{# -- Enable once balances work again
+,ref('labels_trader_portfolios')
```
Comment (aalan3) [inline models/labels/addresses/dex/labels_dex.sql:13]: @jeff-dude note that I disabled this one because it depends on ethereum balances.

Comment (jeff-dude) [reply models/labels/addresses/dex/labels_dex.sql:13]: how do you want to track this (adding back in future)? backlog task in linear?

**Review by jeff-dude — APPROVED** [review]

File: models/labels/addresses/dex/labels_dex.sql (hunk 1 above)
Comment (aalan3) [reply models/labels/addresses/dex/labels_dex.sql:13]: I've created a task to track it

**Pull request merged**